backend/migrations/README.md

```diff
@@ -0,0 +1,84 @@
+# Managing DB with Alembic locally
+
+When we run ```docker-compose up``` the postgres container is created with no tables or schemas.
+
+Upon start of GraphQL container, sqlalchemy ```declarative_base``` is used to create all tables with this function: 
+```Base.metadata.create_all(engine.engine)```. **The number of tables depends on the modules that are enabled in ```config.json``` (in the root of the project).**
+
+As the database is created from scratch, it has no current information about migration state, so, first we need to run database upgrade.
+After that alembic will be able to generate the further migrations localy.
+
+To upgrade database without generating migrations use
```

Comment:
"upgrade database" might lead to confusion, maybe we can write here the sentence "This command will apply all migrations, and syncronize the DB state with local alembic historyt of migrations."